scrapers/Hypnotube.yml

```diff
@@ -0,0 +1,28 @@
+name: "Hypnotube"
+sceneByURL:
+  - action: scrapeXPath
+    url:
+      - hypnotube.com
+    scraper: sceneScraper
+xPathScrapers:
+  sceneScraper:
+    common:
+      $studio: //a[starts-with(@href,'https://hypnotube.com/user/')]
+    scene:
+      Title: //div[@class='item-tr-inner-col inner-col']/h1/text()
+      Details: //meta[@name='description']/@content
```

Comment:
For longer descriptions details are truncated...
We could get the details from the same script that has the image. Some special chars would need to be replaced though. Is it worth it to check that out?

Reply:
Indeed, I should have tested videos with longer description. In my last commit, I went for the actual description in the page (class "main-description"). That way, we don't need to replace special char as we would have to if we were to scrap it from the js.